packages/contracts/contracts/optimistic-ethereum/iOVM/bridge/tokens/iOVM_L1StandardBridge.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >0.5.0;
+pragma experimental ABIEncoderV2;
+
+import './iOVM_L1ERC20Bridge.sol';
+
+/**
+ * @title iOVM_L1StandardBridge
+ */
+interface iOVM_L1StandardBridge is iOVM_L1ERC20Bridge {
+
+    /**********
+     * Events *
+     **********/
+    event ETHDepositInitiated(
```

Comment:
Nit: missing whitespace

Reply:
Really? We have missing whitespaces everywhere in event declarations!